.circleci/config.yml

```diff
@@ -1357,12 +1357,6 @@ workflows:
           parallelism: 8
           requires:
             - Build (nightly)
-      - playwright-functional-tests-chromium:
```

Comment:
Do we need to remove this as well? or do it later as part of The Great Culling of 2025?
https://github.com/mozilla/fxa/blob/7781c7c5cc322e7c6fe11e733473812f3734db07/.circleci/config.yml#L784-L812